### DISCUSSION TITLE
最新版不支持Koolshare 梅林固件路由器，能抢救下吗？😭

### DISCUSSION THREAD

**Discussion by KimiMeta** [messense/aliyundrive-webdav#767] [Q&A]
发现2.0版之后，移出了Koolshare 梅林固件路由器，目前380固件，请问还能抢救下吗？感谢！

**Comment by Nemolovesclaire** [comment]
卖了梅林，换了openwrt..

**Comment by messense** [comment]
发起了一个梅林固件支持的投票：https://github.com/messense/aliyundrive-webdav/discussions/778
